pyodide_build/common.py

```diff
@@ -1,9 +1,18 @@
 from pathlib import Path
 from typing import Optional, Set
+import shutil
 
 ROOTDIR = Path(__file__).parents[1].resolve()
 TOOLSDIR = ROOTDIR / "tools"
-PACKAGERDIR = ROOTDIR / "emsdk" / "emsdk" / "fastcomp" / "emscripten" / "tools"
+
+# Use emcc.py because emcc may be a ccache symlink
+_EMCC_PATH = shutil.which("emcc.py")
+if _EMCC_PATH is None:
+    print("emcc.py not found. Setting file_packager.py path to /dev/null")
+    PACKAGERPATH = Path("/dev/null")
```

Comment:
Shouldn't we rather error here?

Reply:
That would break all pyodide_build tests

Reply:
I could make it a function instead

Reply:
Otherwise maybe keep to to None here, and then in the other place where we need it, since that's not used in test, check it if's None and error. 

I'm just not very keen on the `Path(/dev/null)` because if it's actually used somewhere it will be confusing to debug.